integration/test/ParseQueryTest.js

```diff
@@ -1537,4 +1537,110 @@ describe('Parse Query', () => {
       done();
     });
   });
+
+  it('can perform a full text search with language options', (done) => {
+    const subjects = [
+      'café',
+      'loja de café',
+      'preparando um café',
+      'preparar',
+      'café com leite',
+      'Сырники',
+      'prepare café e creme',
+      'preparação de cafe com leite',
+    ];
+    const objects = [];
+    for (const i in subjects) {
+      const obj = new TestObject({ comment: subjects[i] });
+      objects.push(obj);
+    }
+    Parse.Object.saveAll(objects).then(() => {
+      const q = new Parse.Query(TestObject);
+      q.fullText('comment', 'preparar', { language: "portuguese" });
+      return q.find();
+    }).then((results) => {
+      assert.equal(results.length, 4);
+    }).then(null, () => {
+      done();
```

Comment:
`done()` should be in the `then()` not the error.

Your test are suppose to pass but throwing an error. 

Check all your tests.